File: projects/project 2/source/CameraApp/CameraApp/MainActivity.cs
using System;
using System.Collections.Generic;
using Android.App;
using Android.Content;
using Android.Content.PM;
using Android.Graphics;
using Android.OS;
using Android.Provider;
using Android.Widget;
using Java.IO;
using Environment = Android.OS.Environment;
using Uri = Android.Net.Uri;

namespace CameraApp
{   
    //creating a new static class with variables
    //do no forget to give permission to app to access external memory
    //and camera in manifest
    public static class App
    {
        public static File _file;
        public static File _dir;
        public static Bitmap bitmap;

    }
    [Activity(Label = "CameraApp", MainLauncher = true, Icon = "@mipmap/icon")]
    public class MainActivity : Activity
    {
        int count = 1;


        //after class is created and permissions are given we need to update this OnCreate
        //function 
        protected override void OnCreate(Bundle savedInstanceState)
        {
            base.OnCreate(savedInstanceState);

            // Set our view from the "main" layout resource
            SetContentView(Resource.Layout.Main);

            // Get our button from the layout resource,
            // and attach an event to it
            Button button = FindViewById<Button>(Resource.Id.myButton);

            button.Click += delegate { button.Text = string.Format("{0} clicks!", count++); };


            
            //Good and helpful references to help with camera app.
            //https://developer.xamarin.com/api/type/Android.Hardware.Camera/
            //http://www.c-sharpcorner.com/article/creating-a-camera-app-in-xamarin-android-app-using-visual-studio-2015/

        }
    }
}

File: projects/project 3/source/ImageAnalysis/app/src/main/java/hsu/imageanalysis/MainActivity.java
package hsu.imageanalysis;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

//NOTE: You will need to set an API key in ViewActivity.java

public class MainActivity extends AppCompatActivity
{
    @Override
    protected void onCreate(Bundle savedInstanceState)
    {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Button button_take_picture = (Button) findViewById(R.id.btn_take_picture);

        button_take_picture.setOnClickListener(new View.OnClickListener()
        {
            @Override
            public void onClick(View v) {
                to_second_activity();
            }
        });
    }

    private void to_second_activity()
    {
        Intent intent = new Intent(this, ViewActivity.class);
        startActivity(intent);
    }
}
